tests/Integration/Resource/UsersTest.php

```diff
@@ -167,6 +167,18 @@ public function testCreateUserWithPasswordCredential(): void
         static::assertNull($user);
     }
 
+    public function testExecuteActionsEmail(): void
```

Comment:
I just tested it out locally - calling this endpoint actually requires a few setup steps (setting a user email address, setting up SMTP).

You can leave out the integration test IMO, thanks for your attempt, though 😄